must-gather/Dockerfile

```
@@ -16,19 +16,13 @@ RUN curl --location --output velero.tgz https://github.com/openshift/velero/arch
     tar -xzvf restic.tgz && cd restic-${RESTIC_BRANCH} && \
     CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -mod=mod -ldflags '-extldflags "-static"' -o /restic github.com/restic/restic/cmd/restic && \
     cd .. && rm -rf restic.tgz restic-${RESTIC_BRANCH} && \
-    curl --location --output oc.tgz https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-4.16/openshift-client-linux-${TARGETARCH}-rhel9.tar.gz && \
+    curl --location --output oc.tgz https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/openshift-client-linux-${TARGETARCH}-rhel9.tar.gz && \
     tar -xzvf oc.tgz -C . oc && rm -rf oc.tgz
 
-FROM registry.access.redhat.com/ubi9/go-toolset:latest AS gobuilder
-
-RUN go install -v github.com/google/pprof@latest
-
 FROM registry.access.redhat.com/ubi9-minimal:latest
 
-RUN echo -ne "[centos-9-appstream]\nname = CentOS 9 (RPMs) - AppStream\nbaseurl = https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/\nenabled = 1\ngpgcheck = 0" > /etc/yum.repos.d/centos-9-appstream.repo
-RUN microdnf -y install rsync tar gzip graphviz findutils
+RUN microdnf -y install tar
```

```suggestion
RUN microdnf -y install tar && microdnf clean all
```
https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/building_running_and_managing_containers/assembly_adding-software-to-a-ubi-container_building-running-and-managing-containers#proc_building-ubi-based-images_assembly_adding-software-to-a-ubi-container